sdk/protocol-manager-poc/tests/tests.spec.ts

```diff
@@ -175,21 +177,35 @@ async function createProtocolManagerContext (): Promise<ProtocolManagerContext>
 
 describe('playground', () => {
   let ctx: ProtocolManagerContext
-  let makerPlugin: ProtocolPlugin
-  let sparkPlugin: ProtocolPlugin
+  let makerPlugin: ProtocolPlugin<MakerPoolId>
+  let sparkPlugin: ProtocolPlugin<SparkPoolId>
   beforeAll(async () => {
     ctx = await createProtocolManagerContext()
     makerPlugin = createMakerPlugin(ctx)
     sparkPlugin = createSparkPlugin(ctx)
   })
 
   it('template/maker', async () => {
-    const result = await makerPlugin.getPool(makerPlugin.getPoolId("ETH-A"))
+    // const result = await makerPlugin.getPool(makerPlugin.getPoolId("ETH-A"))
```

Comment:
is this needed?

Reply:
I'll test it on my PR, as can you see this PR is into my branch not into dev.

Same comment to the next.